etna/models/mixins.py

```diff
@@ -582,6 +582,7 @@ def _make_component_predictions(self, ts: TSDataset, prediction_method: Callable
         # TODO: make it work with prediction intervals and context
         target_components_df = prediction_method(self=self._base_model, df=features_df, **kwargs)
         target_components_df["segment"] = segment_column
+        target_components_df["timestamp"] = features_df["timestamp"]
```

Comment:
We can't remove `segment_column` because we drop segments from `features_df`.
And why we should create a copy in `timestamp_column` when we can use timestamps directly from `features_df?

Reply:
For the for the glory of consistency)))